packages/grid/x-data-grid-pro/src/tests/events.DataGridPro.test.tsx

```diff
@@ -107,7 +108,7 @@ describe('<DataGridPro /> - Events Params', () => {
     let cell11;
 
     it('should include the correct params', () => {
-      const handleClick = (params, event) => {
+      const handleClick = (params: GridCellParams, event: React.MouseEvent) => {
```

Comment:
```suggestion
      const handleClick: GridEventListener<'cellClick'> = (params, event) => {
```

Alternatively you can use this type